### PR TITLE
Perform some uncontroversial(?) refactors

### PR DESCRIPTION
**Pull request by chadwhitacre**
Next doc PR, after #35.

### CODE REVIEW

**Comment by chadwhitacre** [comment]
Ready for review, @Changaco.

**Review by Changaco — CHANGES_REQUESTED** [review]
You can't remove the `repr` decode strategy, we use it in both aspen and pando.

**Review by Changaco — COMMENTED** [review]
Just because we can doesn't necessarily mean we should. This change means we have to release a new version of the aspen-jinja package, unless you're planning to integrate the Jinja renderer directly in Aspen?

**Comment by Changaco** [comment]
(I'm going to sleep now.)

File: docs/reference/index.rst [hunk 1]
```diff
@@ -6,9 +6,9 @@ The :py:mod:`aspen` library is divided into three major subpackages: the
 :py:mod:`~aspen.simplates` file format, an HTTP
 :py:mod:`~aspen.request_processor`, and a partial :py:mod:`~aspen.http`
 implementation. In addition, there are supporting modules for
-:py:mod:`~aspen.configuration`, :py:mod:`~aspen.exceptions`, and
```
Comment (whit537) [inline docs/reference/index.rst]: Foo bar

**Comment by chadwhitacre** [comment]
Hmm ... I'm adjusting to the new GitHub PR review UI. Is there no link between your comments and the relevant lines?

**Comment by chadwhitacre** [comment]
> we have to release a new version of the aspen-jinja package

Hmm ... good point. Also `aspen-pystache` and `aspen-tornado`. That's not that hard, is it? :-)

We should at least issue a deprecation warning in 1.0 saying that `aspen.renderers` will go away in 2.0.

> we use it in both aspen and pando.

Sorry ... where?

**Comment by Changaco** [comment]
> Hmm ... I'm adjusting to the new GitHub PR review UI. Is there no link between your comments and the relevant lines?

Apparently not, maybe I'm using it wrong though.

> That's not that hard, is it? :-)

No, but it's still time we wouldn't be spending on more useful stuff.

> We should at least issue a deprecation warning in 1.0 saying that `aspen.renderers` will go away in 2.0.

I'm okay with that.

> Sorry ... where?

`git grep "'repr'"`

**Comment by chadwhitacre** [comment]
> I'm okay with that.

Ready for review in 66b690cbd5a2b9fb1381133cc503db2207b02dc7.

> You can't remove the `repr` decode strategy, we use it in both aspen and pando.

How about efa830a21e38db2c421a722e63745de6ddfeaf44?

**Comment by chadwhitacre** [comment]
I've reworked the doc structure and written a first draft of the dispatch doc. I'm keeping it on the `docs` branch for now. If I get all that written before we land this then I'll maybe make a separate branch.

screen shot 2016-09-17 at 11 31 26 am

**Comment by chadwhitacre** [comment]
I've got a draft of "How to Write a Simplate" and half of "How to Write a Plugin." Going afk for a while.

**Review by Changaco — APPROVED** [review]

**Pull request merged**